genesyscloud/resource_genesyscloud_quality_forms_evaluation.go

```diff
@@ -16,17 +16,25 @@ import (
 	"terraform-provider-genesyscloud/genesyscloud/consistency_checker"
 
 	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
-	lists "terraform-provider-genesyscloud/genesyscloud/util/lists"
+	"terraform-provider-genesyscloud/genesyscloud/util/lists"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	"github.com/mypurecloud/platform-client-sdk-go/v133/platformclientv2"
 )
 
+const readOnlyWarningMessage = "READ-ONLY. No value should be provided for this field as one will be generated by the API."
+
 var (
 	evaluationFormQuestionGroup = &schema.Resource{
 		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: fmt.Sprintf("ID of the question group. %s", readOnlyWarningMessage),
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
I believe you only need to set `Computed: true` and don't need to set `Optional: true` if the user is not intended to set/update this value. See the docs for `Optional` and `Computed` behaviors here: https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors

Reply:
I was trying to find a way to mark an attribute as read-only! Didn't find that article. Thanks, Brian!

Reply:
Woohoo! Glad to help!